_posts/2019-06-06-omero-5-5-0.md

```diff
@@ -0,0 +1,73 @@
+---
+layout: post
+title: Release of OMERO 5.5.0
+intro-blurb: The OME team is pleased to announce the release of OMERO 5.5.0.
+---
+Today we are releasing OMERO 5.5.0, a full production-ready release of OMERO.
+
+This release also upgrades the version of Bio-Formats which OMERO uses to [6.1.0](https://docs.openmicroscopy.org/bio-formats/6.1.0/about/whats-new.html).
+
+This version **does not** require a server upgrade.
+
+For more information about the aim of this 5.5 series, please read
+
+This release focuses on dropping support for Java 7, Python 2.6 and Ice 3.5,
+adding support for Java 11, and on decoupling the Java components to new,
+separate repositories, each with a new [Gradle](https://gradle.org) build system:
+
+- https://github.com/ome/omero-dsl-plugin
+- https://github.com/ome/omero-model
+- https://github.com/ome/omero-common
+- https://github.com/ome/omero-romio
+- https://github.com/ome/omero-renderer
+- https://github.com/ome/omero-server
+- https://github.com/ome/omero-blitz
+- https://github.com/ome/omero-gateway-java
+- https://github.com/ome/omero-blitz-plugin
+- https://github.com/ome/omero-insight
+- https://github.com/ome/omero-matlab
+- https://github.com/ome/omero-javapackager-plugin
+- https://github.com/ome/omero-api-plugin
+
+This has the goal of enabling more fine-grained releases.
+A new restriction is that the names of server configuration properties
```

Comment:
can I suggest putting this under a sysadmin section below? Don't know if @manics has anything else he'd highlight.